Empire/EmpireFirmware/EmpireFirmware.ino

```diff
@@ -92,7 +92,7 @@ uint8_t checksum2 = 0;
 
```

Comment:
See latest commits on main repo. I commited a little prematurely. Before we merge the branches I want to make sure the code in this branch works with the new packet structure I made. Added a from byte and some packet id stuff. We need a way to add a fully constructed packet into a packet buffer to then get sent to the Brooklyn. I currently have a command that adds a byte to the buffer every time you call it and we can just iterate through a packet. But we need to decide how we want to send long response packets. For example the user tells a motor to go to a position the empire will then respond back immediately with a success code and maybe its current position. The main API code then continues on. Then when it finally reaches the desired position the Empire adds a packet to the buffer with the same packet id that was originally sent in the command the API can then read this. The question is do we just make this a mirror packet or do we make another success or error code packet and say okay we made it to the position or throw like a timeout error. I personally like the success error code solution for this as well just need to make sure we remember the packet id for when we send data back.